runtime/src/accounts_db.rs

```diff
@@ -7014,6 +7057,7 @@ impl AccountsDb {
                 storage_size_accounts_map_us: storage_info_timings.storage_size_accounts_map_us,
                 storage_size_accounts_map_flatten_us: storage_info_timings
                     .storage_size_accounts_map_flatten_us,
+                accounts_data_len_dedup_time_us: accounts_data_len_dedup_timer.as_us(),
```

Comment:
@jeffwashington Let me know what you think about this way of handling the timings. Since the timer only runs on `pass == 0`, this value will be 0 for other passes. Is that an issue?

Reply:
0 is fine. pass==1 is only used in a verification/debugging pass with a cli arg.